src/components/HomeView.vue

```diff
@@ -1,6 +1,6 @@
 <template>
     <div id="home-view" class="jumbotron">
-        <h1><img src="../assets/images/logo.png"></h1>
+        <h1><img src="../assets/images/logo.png" class="img-responsive"></h1>
```

Comment:
Looks good!
Could you also add an alt attribute please. 
I'll approve after that.

Reply:
Alright I'm on it